googleapiclient/discovery.py

```diff
@@ -956,6 +956,16 @@ def _add_basic_methods(self, resourceDesc, rootDesc, schema):
       batch_uri = '%s%s' % (
         rootDesc['rootUrl'], rootDesc.get('batchPath', 'batch'))
       def new_batch_http_request(callback=None):
+        """Create a BatchHttpRequest object based on the discovery document.
+
+        Args:
+          callback: callable, A callback to be called for each response, of the
+            form callback(id, response, exception). The first parameter is the
+            request id, and the second is the deserialized response object. The
+            third is an apiclient.errors.HttpError exception object if an HTTP
+            error occurred while processing the request, or None if no error
+            occurred.
```

Comment:
Doesn't this doc string need a "Returns" section since the function returns a value?